### ISSUE TITLE
Attract donors with DEI interests to SAFS

### ISSUE THREAD

**Issue by mdscheuerell** [OARS-SAFS/DEI#63]
Work with the Development office to try and attract wealthy donors to SAFS who are interested in supporting DEI efforts. There are a lot of wealthy people in the Puget Sound region who are interested in philanthropy. Can we work with some of them to help advance funding opportunities for BIPOC students in STEM?

**Comment by marksorel8** [comment]
This is an interesting idea @mdscheuerell. I'm curious to hear more of your thoughts on this. I think we all agree that more BIPOC students should be funded with the streams that are currently going to mostly White students. I'm sure that partnering with new funding sources could move our institution in the right direction, but their interests would need to align with ours.  What I am hearing, and it makes sense, is that diversity grants shouldn't be necessary to increase representation.

**Comment by mdscheuerell** [comment]
> What I am hearing, and it makes sense, is that diversity grants shouldn't be necessary to increase representation.

Yes, @marksorel8, I agree with this sentiment. This idea was meant as merely a suggestion for improving our efforts to recruit and retain BIPOC students. As many students and faculty have noted, student stipends are quite low relative to the cost of living in Seattle. Having sources of money that are intended to directly supplement student stipends or offset other  costs could help us attract a wider pool of potential students.

Some of the potential donors themselves might be BIPOC. For example, Usha Varanasi (former NWFSC director) and her husband Rao established a [fellowship for undergrads in biology](https://www.biology.washington.edu/student-awards/varanasi-scholarship), a [fellowship for grads in SAFS and SMEA](https://environment.uw.edu/news/2014/12/philanthropy-making-a-difference-5/), and the SAFS Research Derby. Given the amount of wealth in the Puget Sound region and an above-average concern for the environment, it seems that we might be able to capitalize on some additional, and perhaps newfound, interest of others for supporting education initiatives for underrepresented minorities.

**Comment by marksorel8** [comment]
I see. That makes brilliant sense @mdscheuerell. Thanks!
